net-mgmt/net-snmp/src/opnsense/scripts/OPNsense/Netsnmp/setup.sh

```diff
@@ -7,3 +7,5 @@ chmod 755 /var/net-snmp
 mkdir -p /var/net-snmp/mib_indexes
 chown -R root:wheel /var/net-snmp/mib_indexes
 chmod 700 /var/net-snmp/mib_indexes
+
+chmod 755 /usr/local/sbin/distro
```

Comment:
```suggestion
```

let me fix that in git after merge which is a bot more elegant and required for scripts content anyway

Reply:
I already fixed it in git. Sorry, forgot to remove that.